src/components/searchBar/index.tsx

```diff
@@ -12,8 +17,32 @@ export const SearchBar = (props: SearchBarProps) => {
       <input
         {...props}
         type="text"
-        className={`h-10 w-full rounded-full border pl-10 pr-3 focus:outline-none ${props.className}`}
+        className={`h-10 w-full rounded-full border pl-10 ${submitButton ? 'pr-[38px]' : 'pr-3'} focus:outline-none ${props.className}`}
+        onSubmit={() => {
+          console.log('howdy from submit');
+        }}
+        onKeyDown={(e) => {
+          if (e.key === 'Enter' && submitLogic) {
```

Comment:
should this be `typeof submitLogic !== 'undefined'`

Reply:
does the same thing but i agree.